packages/main/src/TabContainer.hbs

```diff
@@ -7,15 +7,24 @@
 	{{/if}}
 
 	<div class="{{classes.header}}" id="{{_id}}-header">
+		<div
+				class="ui5-tc__overflowButton ui5-tc__startOverflowButton"
+				@click="{{_onStartOverflowButtonClick}}"
+				hidden
+		>
+			{{#if startOverflowButton.length}}
+				<slot name="startOverflowButton"></slot>
+			{{else}}
+				<ui5-button
+						icon="{{overflowMenuIcon}}"
+						data-ui5-stable="tabs-overflow"
+						tabindex="-1"
+						title="{{overflowMenuTitle}}"
+						aria-haspopup="true"
+						icon-end>{{this._startOverflowText}}</ui5-button>
+			{{/if}}
+		</div>
 		<div class="{{classes.headerInnerContainer}}">
-			<div class="{{classes.headerBackArrow}}">
-				<ui5-button @click="{{_onHeaderBackArrowClick}}"
-							icon="slim-arrow-left"
-							data-ui5-stable="tabs-left"
-							design="Transparent"
-							tabindex="-1"
-							title="{{previousIconACCName}}"></ui5-button>
-			</div>
 			<!-- tab items -->
 			<div class="{{classes.headerScrollContainer}}" id="{{_id}}-headerScrollContainer">
```

Comment:
This shouldn't be called 'headerScrollContainer' any more

Reply:
Done